src/background/js/app.js

```diff
@@ -138,7 +165,7 @@ gApp.run(function ($rootScope, $location, $http, $timeout, ProfileService, Setti
                         }
 
                         $http.get(apiBaseURL + "account").success(function (data) {
-                            $rootScope.profile.user = data;
+                            //$rootScope.profile.user = data;
```

Comment:
Sorry, I just* modified them (and weren't modified when you commented, which pointed out that I actually forgot some, which are fixed now).